globus_folder_sync.py

```diff
@@ -48,6 +49,8 @@
 SCOPES = ('openid email profile '
           'urn:globus:auth:scope:transfer.api.globus.org:all')
 
+APP_NAME = 'My App'
```

Comment:
Let's make this a little more descriptive, like "Globus Folder Sync Example App"